content/pt/docs/tutorials/hello-minikube.md

```diff
@@ -0,0 +1,258 @@
+---
+title: Olá, Minikube!
+content_type: tutorial
+weight: 5
+menu:
+  main:
+    title: "Iniciar"
+    weight: 10
+    post: >
+      <p>Pronto para meter a mão na massa? Vamos criar um cluster Kubernetes simples e executar uma aplicacão exemplo.</p>
+card:
+  name: tutorials
+  weight: 10
+---
+
+<!-- overview -->
+
+Este tutorial mostra como executar uma aplicação exemplo no Kubernetes utilizando o [Minikube](https://minikube.sigs.k8s.io) e o [Katacoda](https://www.katacoda.com). O Katacoda disponibiliza um ambiente Kubernetes gratuito e acessível via navegador. 
+
+{{< note >}}
+Você também consegue seguir os passos desse tutorial instalando o Minikube localmente. Para instruções de instalação, acesse: [iniciando com minikube](https://minikube.sigs.k8s.io/docs/start/).
+{{< /note >}}
+
+## Objetivos
+
+* Instalar uma aplicação exemplo no minikube.
+* Executar a aplicação.
+* Visualizar os logs da aplicação.
+
+## Antes de você iniciar
+
+Este tutorial disponibiliza uma imagem de contêiner que utiliza o NGIX para retornar todas as requisições.
```

Comment:
Hi Jailton, please, can you solve the typo in the word "NGIX" to "NGINX"?
Thanks for your contribution!